docs/manual/installation.rst

```diff
@@ -31,10 +31,10 @@ Then install phpspec with the composer install command:
 
     $ composer install
 
-Follow instructions on `the composer website <https://getcomposer.org/download/`_
+Follow instructions on `the composer website <https://getcomposer.org/download/>`_
 if you don't have it installed yet.
 
-Phpspec with its dependencies will be installed inside the ``vendor`` folder
+phpspec with its dependencies will be installed inside the ``vendor`` folder
```

Comment:
There's no consistent style guide for how to style 'PhpSpec'. Some brands insist on lowercase even at the start of a line, English experts think that's silly.... 

*Sigh*, I'll write a style guide